### PR TITLE
AESinkFactory: Check return value of pulse

### PR DESCRIPTION
**Pull request by fritsch**
This implements a return value for pulse.

Sadly I need pasimple for that, as normally the factory creates us ...

### CODE REVIEW

**Comment by fritsch** [comment]
@wsnipex did I hack the cmake file correctly?

File: cmake/modules/FindPulseAudio.cmake [hunk 1]
```diff
@@ -21,12 +21,16 @@ endif()
...
   pkg_check_modules(PC_PULSEAUDIO_MAINLOOP libpulse-mainloop-glib QUIET)
+  pkg_check_modules(PC_PULSEAUDIO_SIMPLE libpulse-simple QUIET)
 endif()
 
-find_path(PULSEAUDIO_INCLUDE_DIR NAMES pulse/pulseaudio.h
+find_path(PULSEAUDIO_INCLUDE_DIR NAMES pulse/pulseaudio.h pulse/simple.h
                                  PATHS ${PC_PULSEAUDIO_INCLUDEDIR} ${PC_PULSEAUDIO_INCLUDE_DIRS})
 
-find_library(PULSEAUDIO_LIBRARY NAMES pulse libpulse
+find_library(PULSEAUDIO_LIBRARY NAMES pulse libpulse pulse-simple libpulse-simple
```
Comment (wsnipex) [inline cmake/modules/FindPulseAudio.cmake]: undo this please, it is searched for in the check you added below.
otherwise looks good

Comment (fritsch) [reply cmake/modules/FindPulseAudio.cmake]: Thanks much - removed it and squashed it in!

**Comment by fritsch** [comment]
jenkins build this please

**Comment by fritsch** [comment]
I killed the jenkins job related to 13184: Now jenkins build this please

**Review by FernetMenta — APPROVED** [review]

**Pull request merged**